packages/terra-framework-docs/src/terra-dev-site/doc/date-time-picker/example/DateTimePickerTimeZone.jsx

```diff
@@ -21,20 +22,21 @@ function DateTimePickerTimeZone() {
         Selected ISO Date Time:
         {dateTime}
       </p>
-      <p>
-        Initial Timezone:
-        {timeZone}
-      </p>
-      <Field label="Enter Date/Time" htmlFor="timeZone">
-        <DateTimePicker
-          ariaLabel="Enter Date/Time"
-          name="date-time-picker-example"
-          dateInputAttributes={{ id: 'timezone' }}
-          onChange={handleDateTimeChange}
-          initialTimeZone={timeZone}
-          value={dateTime}
-        />
-      </Field>
+      <Fieldset
+        type="checkbox"
```

Comment:
why is type attribute used here?

Reply:
Not needed .Changed https://github.com/cerner/terra-framework/pull/2036/commits/08f942a9a26f89ce517a377222b42091223e96cb